hack/non-olm-install/install-gitops-operator.sh

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/env bash
+
+NAMESPACE_PREFIX=${NAMESPACE_PREFIX:-"gitops-operator-"}
+GIT_REVISION=${GIT_REVISION:-"master"}
+
+# gitops-operator version tagged images
+OPERATOR_REGISTRY=${OPERATOR_REGISTRY:-"registry.redhat.io"}
+GITOPS_OPERATOR_VER=${GITOPS_OPERATOR_VER:-"v1.8.1-1"}
```

Comment:
Looking at the script, its pulling a very specific operator build NVR from the registry, I don't know if there is a latest tag maintained in brew/registry.redhat.io that tracks the latest build no. for the latest z release of the operator 

I think having it be a specific version and build is probably okay since the whole idea is to have fine grained control over what is getting installed or what it is getting rolled back to

Reply:
The latest tag did not work. I wanted to use a default value that works for all. So wanted it to be a publicly available image. Since nightly build images are not available in registry.redhat.io, I am using 1.8.1. This has disadvantage of getting outdated soon. Not sure if there is a better `tag` that gets constantly updated.